.github/workflows/test.yml

```diff
@@ -92,9 +92,11 @@ jobs:
         run: sh .github/workflows/install-deps.sh
 
       - name: Build the package
+        working-directory: analyzer
         run: |
           make pip_dev_deps
           BUILD_UI_DIST=NO make package
+          mv build ../
```

Comment:
I think it is better to test the analyze command by building the main package because most of the users use it. We skip building the UI part here, so it will be fast. So I recommend to not change the code in this file.